subscription-service/src/test/kotlin/com/egm/stellio/subscription/utils/ParsingUtilsTests.kt

```diff
@@ -70,4 +71,29 @@ class ParsingUtilsTests {
             })
         }
     }
+
+    @Test
+    fun `it should not allow a subscription if remote JSON-LD @context cannot be retrieved`() {
+        val contextNonExisting = "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context-non-existing.jsonld"
+        val subscription = mapOf(
+            "id" to "invalidId",
+            "type" to "Subscription",
+            "entities" to listOf(mapOf("type" to BEEHIVE_TYPE)),
+            "notification" to mapOf("endpoint" to mapOf("uri" to "http://my.endpoint/notifiy"))
+        )
+
+        runBlocking {
```

Comment:
no way to use the cleaner `runTest` we typically use?

Reply:
yeah I do it but another test don't use `runTest`. Am I standardizing everything?